index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Lymon Zerga's Lemons | Used "cars" for sale</title>
+	<link href="https://fonts.googleapis.com/css?family=Muli:300,400" rel="stylesheet">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<div class="container-fluid">
+		<nav class="navbar navbar-default">
+			<div class="container-fluid">
+			  <!-- Brand and toggle get grouped for better mobile display -->
+			  <div class="navbar-header">
+			    <a class="navbar-brand" href="#">Sick cars, right?</a>
+			  </div>
+
+			  <!-- Collect the nav links, forms, and other content for toggling -->
+			  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+			    <form class="navbar-form navbar-left" role="search">
+			      <div class="form-group">
+			        <input type="text" class="form-control" id="inputBox" placeholder="Search">
+			      </div>
+			      <button type="submit" class="btn btn-default" id="submitBtn">Submit</button>
+			    </form>
```

Comment:
You're just about there, the first issue I found has to do with the search-form in the navbar. Is it is, when I click on a card and start typing it replaces the description (which is perfect, what we want). However, if I press enter it's supposed  to clear the input bar, instead it reloads the whole page (submits the 'form'). Also, if you click on one card, edit it, then click on the next card; both cards are edited. You'll want to change how your event listeners are attached.